clients/cobol-lsp-vscode-extension/src/__tests__/EncodingIconvTest.spec.ts

```diff
@@ -0,0 +1,15 @@
+import * as iconv from "iconv-lite";
+
+describe('Test iconv lib correctly decodes a binary buffer', () => {
+
+    it('checks iconv-lite decodes correctly for patched ebcdic encodings 1147 / 037/ 1047', () => {
+        expect(iconv.decode(Buffer.from([0xF3, 0xF0, 0xF0, 0x9F, 0x40, 0x40]), "cp1147")).toBe("300€  ");
```

Comment:
`Encoding not recognized: 'cp1147' (searched as: 'cp1147')`

Reply:
Yes. It's because the PR Jenkins file is not picked.
Anton tested this on https://ci.eclipse.org/che4z/job/LSP%20for%20COBOL/job/bugfix%252FUS798758-uss/